econml/tests/test_dml.py

```diff
@@ -171,7 +180,8 @@ def make_random(n, is_discrete, d):
                                             eff = est.effect(X, T0=T0, T1=T)
                                             self.assertEqual(shape(eff), effect_shape)
 
-                                            if isinstance(est, LinearDMLCateEstimator) or\
+                                            if isinstance(est, DMLCateEstimator) or\
+                                                    isinstance(est, LinearDMLCateEstimator) or\
                                                     isinstance(est, SparseLinearDMLCateEstimator):
```

Comment:
Minor refactoring: `isinstance`'s second argument can be a tuple, so we can get rid of the `or`s:
```suggestion
                                            if isinstance(est, (DMLCateEstimator, LinearDMLCateEstimator, SparseLinearDMLCateEstimator)):
```